packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -66,4 +67,112 @@ export interface ICheckboxProps extends React.HTMLAttributes<HTMLElement | HTMLI
    * @default 'start'
    */
   boxSide?: 'start' | 'end';
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
```

Comment:
Hm looks like dzearing may have changed it while I wasn't looking. Leave it in for now and I will bring it up with dzearing some other time.